pages.de/common/nix.md

```diff
@@ -1,24 +1,34 @@
 # nix
 
-> Dienstprogramme für die Nix-Sprache und den Nix-Speicher.
+> Ein leistungsfähiger Paketmanager, der das Paketmanagement zuverlässig, reproduzierbar und deklarativ macht.
+> `nix` ist experimentell und muss gesondert aktiviert werden. Für die klassische, stabile Schnittstelle siehe `tldr nix classic`.
+> Einige Unterbefehle wie `build`, `develop`, `flake`, `registry`, `profile`, `search`, `repl`, `store`, `edit`, `why-depends` usw. haben ihre eigene Benutzerdokumentation.
 > Weitere Informationen: <https://nix.dev/manual/nix/stable/command-ref/new-cli/nix>.
 
-- Suche nach einem Paket über seinen Namen oder seine Beschreibung:
+- Aktiviere den `nix` Befehl:
 
-`nix search {{suchbegriff}}`
+`mkdir {{[-p|--parents]}} ~/.config/nix; echo 'experimental-features = nix-command flakes' > ~/.config/nix/nix.conf`
 
-- Starte eine Nix-Shell, die die angegebenen Pakete zur Verfügung stellt:
+- Suche nach einem Paket in nixpkgs via Packagenamen oder Beschreibung:
 
-`nix run {{nixpkgs.pkg1 nixpkgs.pkg2 ...}}`
+`nix search nixpkgs {{suchbegriff}}`
 
-- Optimiere die Festplattennutzung des Nix-Speichers durch Zusammenfassen doppelter Dateien:
+- Starte eine Nix-Shell und stelle die angegebenen Pakete von nixpkgs darin bereit:
```

Comment:
I don't know a lot about Nix. The English version sounds to me like it is starting a shell (`bash`/`zsh`/`fish`/etc). Is that the case or is your description more precise?